tools/pythonpkg/tests/fast/pandas/test_df_object_resolution.py

```diff
@@ -529,17 +529,14 @@ def test_double_object_conversion(self, pandas, duckdb_cursor):
 
     @pytest.mark.parametrize('pandas', [NumpyPandas(), ArrowPandas()])
     def test_numpy_object_with_stride(self, pandas, duckdb_cursor):
-        df = pandas.DataFrame(columns=["idx", "evens", "zeros"])
-
-        df["idx"] = list(range(10))
-        for col in df.columns[1:]:
-            df[col].values[:] = 0
```

Comment:
Without this change a deprecation warning is elicited from the fact that the Pyodide version of pandas is a bit outdated.

Reply:
Please make sure that this has the same behavior as before, we don't have many tests for strides and I would be afraid of a regression there if this test were to silently break

Reply:
I'll probably just revert my changes for now.

Reply:
Reverted